cmd/podman/start.go

```diff
@@ -132,6 +134,18 @@ func startCmd(c *cli.Context) error {
 		}
 		// Handle non-attach start
 		if err := ctr.Start(ctx); err != nil {
+			var createArtifact cc.CreateConfig
```

Comment:
Do we support `--rm` with `start`? I think we only allow it on `podman run` and not create, in which case we can drop this

Reply:
```
$ sudo podman create --help | grep "\-\-rm"
   --rm                                          Remove container (and pod if created) after exit
```

Reply:
you didnt really think i would do more work than needed did you?